feat: Add speed control variable (#81)
DenverCoder1/readme-typing-svg: README.md

|  `center`   |   `true` to center text or `false` for left aligned (default: `false`)       | boolean |          `true` or `false`          |
|  `vCenter`  | `true` to center vertically or `false`(default) to align above the center    | boolean |          `true` or `false`          |
|  `height`   |            Height of the output SVG in pixels (default: `50`)                | integer |         Any positive number         |
|   `width`   |            Width of the output SVG in pixels (default: `400`)                | integer |         Any positive number         |
|   `font`    |                    Font family (default: `monospace`)                        | string  |     Any font from Google Fonts      |
|   `size`    |                   Font size in pixels (default: `20`)                        | integer |         Any positive number         |
|   `color`   |                  Color of the text (default: `36BCF7`)                       | string  |  Hex code without # (eg. `00ff00`)  |
| `multiline` | `true` to wrap lines or `false` to retype on one line (default: `false`)     | boolean |          `true` or `false`          |

> |  `duration` | Duration of the printing of output SVG in milliseconds (default: `5000`)     | integer |         Any positive number         |

Reviewer: Maybe make it more clear that it is per line?
```suggestion
|  `duration` | Duration of printing a single line of output in milliseconds (default: `5000`) | integer |         Any positive number         |
```
